fable-test-app/src/pages/About.tsx

```diff
@@ -1,13 +1,62 @@
 import React from 'react';
 
 // Components (internal)
-import { DateModified, Heading } from '../components';
+import { DateModified, Heading, Text } from '../components';
+import { GcdsCard, GcdsGrid } from '@cdssnc/gcds-components-react';
+import docsImage from '../assets/docs.png';
+import figmaImage from '../assets/figma.png';
+import githubImage from '../assets/github.png';
```

Comment:
If you add the images in the `public` folder instead of `src/assets`, you don't have to import them.